lib/main.js

```
@@ -43,31 +53,31 @@ if (externalReferencedNamespaces.length > 0) {
+    const m = require(moduleName);
     delete namespaceRegistry[namespace];
     namespaceRegistry[namespace] = m;
     return m;
   }
 
-  for (var i in externalReferencedNamespaces) {
-    var ns = externalReferencedNamespaces[i];
+  for (const i in externalReferencedNamespaces) {
+    const ns = externalReferencedNamespaces[i];
 
     if (!namespaceRegistry.hasOwnProperty(ns)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)